server/app/services/DateConverter.java

```diff
@@ -61,10 +61,19 @@ public Instant parseIso8601DateToStartOfDateInstant(String dateString) {
     return parseIso8601DateToLocalDate(dateString).atStartOfDay(zoneId).toInstant();
   }
 
-  /** Formats an {@link Instant} to a date and time in the local time zone. */
+  /** Formats an {@link Instant} to a human-readable date and time in the local time zone. */
   public String renderDateTime(Instant time) {
     ZonedDateTime dateTime = time.atZone(zoneId);
-    return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd 'at' h:mm a z"));
+    return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd 'at' h:m a z"));
+  }
+
+  /**
+   * Formats an {@link Instant} to a date and time in the local time zone for the purpose of data
+   * exports (with no filler words).
+   */
+  public String renderDateTimeDataOnly(Instant time) {
+    ZonedDateTime dateTime = time.atZone(zoneId);
+    return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd h:m:s a z"));
```

Comment:
I think the minute and second need to be mm and ss so they have the 0 prepended for single digits.

Reply:
Good catch